src/uu/comm/src/comm.rs

```diff
@@ -60,7 +70,71 @@ impl LineReader {
     }
 }
 
-fn comm(a: &mut LineReader, b: &mut LineReader, delim: &str, opts: &ArgMatches) {
+impl OrderChecker {
+    fn new(file_num: usize, check_order: bool) -> Self {
```

Comment:
As there are only two possible values for `file_num`, `1` and `2`, it might make sense to use an enum instead of `usize`.

Reply:
great idea, thanks